code/modules/surgery/eye_cybernetic_implant.dm

```diff
@@ -0,0 +1,31 @@
+/datum/surgery/eye_cybernetic_implant
+	name = "eye cybernetic implant"
+	steps = list(/datum/surgery_step/incise, /datum/surgery_step/clamp_bleeders, /datum/surgery_step/retract_skin, /datum/surgery_step/cybernetic_implant, /datum/surgery_step/fix_eyes, /datum/surgery_step/close)
+	location = "eyes"
+
+/datum/surgery_step/cybernetic_implant
+	accept_any_item = 1
```

Comment:
This step would probably be better off by having it's tool list contain the base type path (/obj/item/organ/cybernetic_implant/eyes) instead of simply accepting any item and doing the type check yourself.